ci-operator/step-registry/storage/conf/csi-kubevirt/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+- davidvossel
+- nirarg
+- nunnatsa
+- qinqon
+- orenc1
+- LiangquanLi930
```

Comment:
add yourself here too

Reply:
Don't think I can, since I am not part of the org yet.